packages/store/test/test-encodeKey.js

```diff
@@ -55,15 +55,23 @@ const getNaN = (hexEncoding = '0008000000000000') => {
 const NegativeNaN = getNaN('ffffffffffffffff');
 
 const goldenPairs = harden([
-  [37n, 'p0000000002:37'],
-  [-1n, 'n9999999999:9'],
   [1, 'fbff0000000000000'],
   [-1, 'f400fffffffffffff'],
   [NaN, 'ffff8000000000000'],
   [NegativeNaN, 'ffff8000000000000'],
   [0, 'f8000000000000000'],
   [Infinity, 'ffff0000000000000'],
   [-Infinity, 'f000fffffffffffff'],
+  [-1234567890n, 'n#89:8765432109'],
+  [-123456789n, 'n0:876543210'],
+  [-1000n, 'n5:8999'],
+  [-999n, 'n6:000'],
```

Comment:
In my ignorance, I have no opinion. What are the pros and cons?

Reply:
Basically just subjective ease vs. difficulty of decoding negative numbers as a human. This encoding uses only 90% of all n-digit sequences (because the remaining 10% start with zero and are therefore actually sequences numbers of _fewer_ than n digits), which can therefore be offset according to taste.

n | nines' complement | ten's complement
-- | -- | --
-1n | n8:8 | n9:9
-9n | n8:0 | n9:1
-10n | n7:89 | n8:90
-11n | n7:88 | n8:89
-99n | n7:00 | n8:01
-100n | n6:899 | n7:900
-900n | n6:099 | n7:100
-999n | n6:000 | n7:001
-1234567890n | n#89:8765432109 | n#90:8765432110

Reply:
10s complement looks easier to me to visually decode. If you agree, let's do that.

Reply:
Done.